Migrating_to_V2.md

```diff
@@ -0,0 +1,107 @@
+# Migration from 1.0 to 2.0
+
+Version 2 of the Amazon Chime React Component Library makes a small number of interface changes, as well as removing some deprecated interfaces. Version 2 of the library also requires a dependency to version 2 of the amazon-chime-sdk-js dependency.
+
+In many cases you should not need to adjust your application code at all. This will be the case if:
+
+- You do not use the `ChatBubble` component or the `ChatBubbleContainer` component.
+- You do not use the `Roster` component.
+- You do not use the `Textarea` component.
+- You do not directly use any APIs from the Amazon Chime SDK. If you do, you should read this migration guide (https://aws.github.io/amazon-chime-sdk-js/modules/migrationto_2_0.html).
+
+If you do, read on.
+
+## Installation
+
+Installation involves adjusting your package.json to depend on version 2.x.x.
+
+```
+npm install --save amazon-chime-sdk-component-library-react@2
+```
+
+## What’s New
+
+- Updated interface for `ChatBubble` component.
+- Using `forwardRef` in `ChatBubbleContainer` component.
+- Using `forwardRef` in `Textarea` component.
+- New `useDevicePermissionStatus` hook exported.
+- A callback parameter has been added to `realtimeUnsubscribeFromVolumeIndicator` hook.
+- Add click handler and image preview to `MessageAttachment` component.
+- Add optional `badge` prop to icon `Button`.
```

Comment:
Can you call out that the js-sdk dependency has been updated to only support 2.x or w/e it is?

Reply:
Great call out